### PR TITLE
add abstract_lef controls to OpenROAD

### PR DESCRIPTION
**Pull request by gadfort**
adds the flags needed to be able to control the abstract lef writing to the OpenROAD driver

### CODE REVIEW

File: siliconcompiler/tools/openroad/scripts/sc_export.tcl [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Write LEF
-write_abstract_lef "outputs/${sc_design}.lef"
+set lef_args []
+if { [lindex [dict get $sc_cfg tool $sc_tool task $sc_task {var} ord_abstract_lef_bloat_layers] 0] == "true" } {
+  lappend lef_args "-bloat_occupied_layers"
+} else {
```
Comment (troibe) [inline siliconcompiler/tools/openroad/scripts/sc_export.tcl:5]: Not familiar with the semantics...
Are `-bloat_occupied_layers` and `-bloat_factor` mutually exclusive?

File: siliconcompiler/tools/openroad/openroad.py [hunk 2]
```diff
@@ -912,6 +912,16 @@ def _define_ord_params(chip):
              'list of files to use for specifying global connections',
              field='help')
 
+    _set_parameter(chip, param_key='ord_abstract_lef_bloat_factor',
+                   default_value='10',
```
Comment (troibe) [inline siliconcompiler/tools/openroad/openroad.py:916]: What does this default value translate to?
Is it the same default that openroad had before?

**Review by troibe — APPROVED** [review]
Looks like it matches openroads parameters.

File: siliconcompiler/tools/openroad/openroad.py (hunk 2 above)
Comment (gadfort) [reply siliconcompiler/tools/openroad/openroad.py:916]: it's the same ad the OpenROAD default. It's the number routing pitches the abstract generator will bloat the shapes by to attempt to simplify the layout. Unfortunately that is not always perfect and leaves some complex shapes behind.

**Pull request merged**